src/widgets/refdata/list-ui.vue

```diff
@@ -2,35 +2,21 @@
 <template>
   <div row>
     <!-- ElasticSearch widget -->
-    <b-form-group :class="isRequired ? 'required' : ''" id="fundertype-form-group" horizontal :label-cols="uiLabel ? labelCols : 1" :description="uiDescription" :label="uiLabel">
-	  <b-input-group>
+    <b-form-group class="list-ui-form-group" :class="isRequired ? 'required' : ''" :label-cols="uiLabel ? labelCols : 1" :description="uiDescription" :label="uiLabel">
```

Comment:
do not remove id field as that enables testing. renaming is ok.

Reply:
renamed